src/inversify.config.ts

```diff
@@ -127,4 +127,5 @@ export interface ArgumentsProvider {
   uri: string;
   auth?: string;
   json?: boolean;
+  fail?: string;
```

Comment:
```suggestion
  fail?: PracticeImpact | "all";
```
I'd set directly the impact type here.